### PR TITLE
Cs.: show string at current address

### PR DESCRIPTION
**Pull request by kazarmy**

### CODE REVIEW

File: libr/core/cmd_meta.c [hunk 1]
```diff
@@ -575,6 +576,19 @@ static int cmd_meta_hsdmf(RCore *core, const char *input) {
 			free (comment);
 		}
 		break;
+	case '.':
+		str = r_meta_get_string (core->anal, type, addr);
+		if (str) {
+			if (type == 's') {
+				char *esc_str = r_str_escape_latin1 (str, false, true);
+				r_cons_printf ("\"%s\"\n", esc_str);
```
Comment (radare) [inline libr/core/cmd_meta.c]: check for null

**Comment by radare** [comment]
looks red to me

File: libr/core/cmd_meta.c (hunk 1 above)
Comment (kazarmy) [reply libr/core/cmd_meta.c]: I always thought that r2 has undefined behavior if it runs out of memory, but see #8680 for a possible solution.

**Comment by radare** [comment]
Nope its not. R2 aims to not crash under any situation.

**Pull request merged**